src/ops/inventory/ec2inventory.py

```diff
@@ -99,20 +101,17 @@ def get_instances_by_region(self, region):
 
             # sort the instance based on name and index, in this order
             def sort_key(instance):
-                components = instance.tags.get('Name', '').rsplit('-', 1)
-                if len(components) == 2:
-                    return (components[0], int(components[1]) if components[1].isdigit() else 0)
-                else:
-                    return components[0]
+                name = instance.tags.get('Name', '')
+                return name + "-" + str(instance.id)
```

Comment:
This seems to be different functionality.

Reply:
Yes, the previous logic was buggy. For some values it was returning a `touple` (`(components[0], int(components[1])`) and for others it was returning a `str` (`components[0]`). Python3 was not able to compare a touple with a str ^_^

Reply:
I would format this using f strings